umake/frameworks/ide.py

```diff
@@ -673,7 +675,8 @@ def __init__(self, category):
                          desktop_filename="visual-studio-code.desktop",
                          required_files_path=["Code"],
                          dir_to_decompress_in_tarball="VSCode-linux-*",
-                         packages_requirements=["libgtk2.0-0"])
+                         packages_requirements=["libgtk2.0-0"],
+                         add_link=True)
```

Comment:
I would really just condition the add_link to the existence of desktop_filename and that's it. Same when removing, if we have a desktop_filename, we remove if exists.